docs/examples/schema_extra_callable.py

```diff
@@ -0,0 +1,16 @@
+# output-json
+from typing import Dict, Any
+from pydantic import BaseModel
+
+def remove_titles(schema: Dict[str, Any]) -> None:
+    for prop in schema.get('properties', {}).values():
+        prop.pop('title', None)
+
+class Person(BaseModel):
+    name: str
+    age: int
+
+    class Config:
+        schema_extra = remove_titles
```

Comment:
I would guess clearer to have something like
```suggestion
        @staticmethod
        def schema_extra(schema: Dict[str, Any]) -> None:
            for prop in schema.get('properties', {}).values():
                prop.pop('title', None)
```

Reply:
might be good to change the test too to confirm this works.